src/index.html

```diff
@@ -4,9 +4,385 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+        src="./images/logo.png" 
+        alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a 
+            href="#" 
+            class="nav__link is-active"
+            >
+            apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+            class="nav__link" 
+            href="#"
+            data-qa="nav-hover"
+            >
+            laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+
+
+    <div class="catalog">
```

Comment:
Do you really need so much empty space between these tags?)

Reply:
done